Assets/LeapMotion/Scripts/LeapImageRetriever.cs

```diff
@@ -119,12 +121,9 @@ public class LeapImageRetriever : MonoBehaviour
 
   private LM_DEVICE GetDevice(int width)
   {
```

Comment:
@GabrielHare I wanted to specifically call this change out. Gets rid of one of the warnings in the Image Retriever.